Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -164,6 +172,7 @@ def module_test_command(client: Client, args):
                 FeedIndicatorType.Account,
                 FeedIndicatorType.CVE,
                 FeedIndicatorType.Domain,
+                FeedIndicatorType.DomainGlob,
                 FeedIndicatorType.Email,
                 FeedIndicatorType.File,
                 FeedIndicatorType.MD5,
```

Comment:
Why are we maintaining a list of the supported values here? Let's make it an attribute on the `FeedIndicatorType` class itself.

Reply:
Not my decision nor implementation I assume that the reason is that this is a subset of indicators, not all from CSP.

Reply:
This is the CSV module. It's meant to be used for specific implementations so by definition it has to be broad - aka it is all types and if it isn't, it should be. This should be moved to CSP.